File: client/src/model/Project.ts
interface IProject {
	id: number
	name: string
	baseSeries: string
	bias: number
}

class Project {
	id: number
	name: string
	baseSeries: string
	bias: number

	constructor(obj?: IProject) {
		this.id = (obj && obj.id) || 0
		this.name = (obj && obj.name) || ''
		this.baseSeries = (obj && obj.baseSeries) || ''
		this.bias = (obj && obj.bias) || 0
	}
}

export default Project

File: server/DocumentsKM/Models/Project.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DocumentsKM.Models
{
    // Проект
    public class Project
    {
        [Key]
        public Int32 Id { get; set; }

        // Название
        [Required]
        [MaxLength(550)]
        public string Name { get; set; }

        // Базовая серия
        [Required]
        [MaxLength(20)]
        public string BaseSeries { get; set; }

        // Смещение основной надписи
        [Required]
        public Int16 Bias { get; set; }
    }
}
